task/04-arrays-tasks.js

```diff
@@ -286,10 +289,9 @@ function propagateItemsByPositionIndex(arr) {
  *   [ 10, 10, 10, 10 ] => [ 10, 10, 10 ]
  */
 function get3TopItems(arr) {
-   throw new Error('Not implemented');
+    return arr.reverse().slice(0, 3);
```

Comment:
It will not work when initial array is not sorted (future test cases).
Please, add sort method